docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1117,7 +1113,8 @@ function setAudioInputDevice() {
     getMediaStreams({sendAudio, receiveAudio}, {audio})
       .then(({localStream}) => {
         if (isMultistream) {
-          return meeting.media.publishTrack(localStream.getAudioTracks()?.[0]);
+          // ignoring sendAudio value, because it cannot be false for multistream (the UI elements from toggleSourcesMediaDirection are disabled)
+          return meeting.publishTracks({microphone: localStream.getAudioTracks()?.[0]});
```

Comment:
seeing a possibility that publishTracks can throw error, do we want to wrap this with try catch? or is it the client which uses this method?